apps/src/studio/levels.js

```diff
@@ -2528,3 +2525,65 @@ levels.js_hoc2015_event_free = {
     }
   ],
 };
+
+levels.hoc2015_blockly_1 = utils.extend(levels.js_hoc2015_move_right,  {
+  editCode: false,
+  startBlocks: whenRunMoveEast,
+  toolbox: tb(hocMoveNSEW),
+  requiredBlocks: [
+    moveRequiredBlock({ dir: '2' }), // East
```

Comment:
Because they are string. Will work on improving this further once we settle on NSEW or UDLR